docs/documentation/updating-the-kit.md

```diff
@@ -1,129 +1,44 @@
-# Updating the kit
+# Updating to the latest version
 
-**Important note**
+1. Make a backup copy of your prototype folder.
 
-If you have made any changes outside the `app` folder, this process will destroy those changes. We will try and improve the update process to avoid this, but in the meantime you will need to make a note of your changes outside `app`, and add them back after updating.
+2. In your prototype, delete everything except the `app` and `.git` folders.
 
-## Steps
+   You may not be able to see the `.git` folder because files and folders that start with `.` are hidden by default.
```

Comment:
This doesn't tell the user what to do if this happens. I think we should be okay to say that if they can't see the `.git` folder, they don't have to do anything because they shouldn't be able to delete an invisible folder from a file explorer. 

(It doesn't seem likely they would use the terminal to remove the files as they're not deleting all of them - but if they did, they would have to add flags to stop also `.app` being deleted - and if you can do that, you should know how to handle this step without deleting `.git`.)

Reply:
Good point, and Joe and I had a discussion about it. We felt that if they can't see the folder, the clear implication is 'you don't need to worry about it' - because we mentioned leaving it alone anyway. This is an assumption we'll watch out for during testing though.

Reply:
To me the wording is inconclusive. I feel like the user might be left hanging and thinking: if I can't see it, do I need to make sure it's there anyway? 
But let's see what comes back from testing 👍